docs/adr/0005-controller-structure.md

```diff
@@ -0,0 +1,124 @@
+# 5. Controller Structure
```

Comment:
Sorry but you'll have to renumber this to `6`, this PR reminded me that I need to finish mine, but it's numbered `5`. ;)

Reply:
Done https://github.com/DFE-Digital/teacher-training-api/pull/1535/commits/2470361600d37f9a5c18de52a10bdb0838be2b88